tests/test_read_messages.py

```diff
@@ -180,14 +151,89 @@ async def test_read_bytes_message():
 
     await nsq.rdy(1)
     message = await nsq.message_queue.get()
-    assert not message.is_processed
+    assert message.can_be_processed
 
     assert message.body == b"\xa1"
     with pytest.raises(UnicodeDecodeError):
         str(message)
 
     await message.fin()
-    assert message.is_processed
+    assert not message.can_be_processed
+
+    await nsq.close()
+    assert nsq.is_closed
+
+
+@pytest.mark.asyncio
+async def test_timeout_messages():
+    nsq = await open_connection()
+    assert nsq.status.is_connected
+
+    first_message = "first test message at " + str(time())
+    first_message_response = await nsq.pub(
+        "test_read_timed_out_messages", first_message,
+    )
+    assert first_message_response.is_ok
+
+    second_message = "second test message at " + str(time())
+    second_message_response = await nsq.pub(
+        "test_read_timed_out_messages", second_message,
+    )
+    assert second_message_response.is_ok
+
+    await nsq.subscribe("test_read_timed_out_messages", "channel1", 1)
+    assert nsq.is_subscribed
+
+    # We need to wait while the message will be timed out.
+    # The default message timeout is 60 secs
+    await asyncio.sleep(61)
```

Comment:
Waiting `61` seconds for a test seems too long. We could configure `nsqd` service to decrease timeout or set up a new one. What do you think?

Reply:
Then tests on the local machine will not pass without nsqd configuration =)

Reply:
`docker-compose.yml` with a prepared environment for integration tests would help

Reply:
Thanks. I'll try.

Reply:
Addressed in ee68139a133ecced69b4735b1a6287514f063d14